backend/src/govsg/services/govsg-callback.service.ts

```diff
@@ -226,6 +283,42 @@ const parseTemplateMessageWebhook = async (
       void govsgMessage?.update(fieldOpts, whereOpts)
       void govsgMessageTransactional?.update(fieldOpts, whereOpts)
       void govsgOp?.update(fieldOpts, whereOpts)
+      // TODO: Add logging statement here for traceability
+      if (!govsgMessage && !govsgOp) {
```

Comment:
Do we need to throw an error?

Reply:
I can throw, but I will need to handle it.

I suppose I should add a log statement here, if this scenario is really unexpected.

We shouldn't throw errors in response to the webhook request as our failure to process data is not their concern.

[Linear](https://linear.app/ogp/issue/SGC-171/add-logging-to-if-govsgmessage-andand-govsgop)

Reply:
Yup, I think adding a log would be sufficient. Thanks!